src/main/java/kr/sesac/aoao/server/item/controller/dto/UseItemNumResponse.java

```diff
@@ -0,0 +1,19 @@
+package kr.sesac.aoao.server.item.controller.dto;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+public class UseItemNumResponse {
+	private final Long item_id;
+	private final Long user_id;
+	private final int item_num;
```

Comment:
```suggestion
	private final Long itemId;
	private final Long userId;
	private final int itemNum;
```
domain도 모두 camel방식이니 snake 형식에서 camel 형식로 변경해주세용 ㅎㅎ